lib/ansible/runner.rb

```diff
@@ -221,6 +221,8 @@ def run_via_cli(hosts, credentials, env_vars, extra_vars, tags: nil, ansible_run
 
         params = runner_params(base_dir, ansible_runner_method, playbook_or_role_args, verbosity)
 
+        # puts "#{env_vars_hash.map { |k, v| "#{k}=#{v}" }.join(" ")} #{AwesomeSpawn.build_command_line("ansible-runner", params)}"
```

Comment:
This is a debug thing that I do _ALL THE TIME_. I'm just leaving it in the code now :)